cargo-dist/src/backend/ci/github.rs

```diff
@@ -671,7 +671,14 @@ fn github_runner_for_target(
     // where random system dependencies can creep in and be very
     // recent. This helps with portability!
     let result = Some(match target_triple.operating_system {
-        OperatingSystem::Linux => runner_to_config(GithubRunnerRef::from_str("ubuntu-22.04")),
+        OperatingSystem::Linux => {
+            // Use ARM-specific runner for aarch64 Linux targets
+            if target_triple.architecture == Architecture::Aarch64 {
```

Comment:
`Architecture::Aarch64` (which [comes from target_lexicon](https://docs.rs/target-lexicon/0.13.2/target_lexicon/enum.Architecture.html)) takes an architecture argument which is necessary here for the `==` comparison to work. Probably easier to do this with a `matches!()` check so you can check against any subarchitecture.
```suggestion
            if matches!(target_triple.architecture, Architecture::Aarch64(_)) {
```

Reply:
Thanks! Rebased and pushed.